apps/plato_single_variant/Readme.md

```diff
@@ -65,15 +62,21 @@ It is reccomended to use this option when running PheWAS as it will exponentiall
 ### Advanced Options
 **`WARNING: If used, leave above "Regression Options" EMPTY. Please refer to documentation in PLATO manual prior to using this option.`**
 
-**PLATO command-line options**: Command-line regression and global options that will be supplied directly to the PLATO. For example, `logistic`, `linear`, `--permutation`, `--interactions`.
-
-Here are some use cases for this option:
-- **Permutation**:
-```logistic --phewas --covariates sex,age,pc1,pc2,pc3,pc4 --permutations 10000 --output out.txt```
-- **Interactions**:
-```logistic --interaction --pairwise --output out.txt```
-- **Genomic Inflation Factor**: This option enables the calculation and reporting of the genomic inflation factor (GIF).
-```logistic --inflation --inflation-method GC --output out.txt```
+**PLATO command-line options**: Command-line regression and global options that will be supplied directly to the PLATO. For example, you can run analyses such as `logistic` and `linear` with options `--permutation` or `--interactions`.
+
+Here are some examples of the Advanced options provided by PLATO:
+  - **Permutation**:
+  ```
+  logistic --phewas --covariates sex,age,pc1,pc2,pc3,pc4 --permutations 10000 --output out.txt
+  ```
+  - **Interactions**:
+  ```
+  logistic --interaction --pairwise --output out.txt
+  ```
+  - **Genomic Inflation Factor**: This option enables the calculation and reporting of the genomic inflation factor (GIF).
+  ```
+  logistic --inflation --inflation-method GC --output out.txt
+  ```
 
 User don't need to specify `load-data` or `load-trait` commands here since app still automate that step and provide those input file to PLATO
```

Comment:
Users don't need to specify `load-data` or `load-trait` commands here since apps still automate that step and provide those input files to PLATO.